tensorflow/core/ops/math_ops.cc

```diff
@@ -478,6 +478,14 @@ REGISTER_OP("Ceil")
 Returns element-wise smallest integer in not less than x.
 )doc");
 
+REGISTER_OP("Rint")
+    .Input("x: T")
+    .Output("y: T")
+    .Attr("T: {float, double}")
+    .Doc(R"doc(
+Returns element-wise integer closest to x.
```

Comment:
If we do adjust the flags, it should happen right here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/lib/core/threadpool.cc#L52

There is no need to save and reset the flags, since it will be thread local.  You'll need to do something similar to ScopedFlushDenormal.